pkg/ruler/base/ruler.go

```diff
@@ -68,6 +68,11 @@ const (
 
 	// errors
 	errListAllUser = "unable to list the ruler users"
+
+	// Alertmanager default values
+	AlertmanagerRefreshIntervalDefault = 1 * time.Minute
+	NotificationQueueCapacityDefault   = 10000
+	NotificationTimeoutDefault         = 10 * time.Second
```

Comment:
It should be possible to make these private (added a common prefix for easier grouping):

```suggestion
	alertmanagerRefreshIntervalDefault             = 1 * time.Minute
	alertmanagerNotificationQueueCapacityDefault   = 10000
	alertmanagerNotificationTimeoutDefault         = 10 * time.Second
```